cookbook/basic_examples/log.py

```diff
@@ -0,0 +1,89 @@
+"""A basic example on how to log the data from a prompt and a chat completion."""
```

Comment:
is it worth adding additional information here around setting up the database etc.?

Reply:
this is a super basic setup that i imagine is self explanatory